consensus/src/state_computer.rs

```diff
@@ -44,7 +54,7 @@ pub struct ExecutionProxy {
     state_sync_notifier: Arc<dyn ConsensusNotificationSender>,
     async_state_sync_notifier: aptos_channels::Sender<NotificationType>,
     validators: Mutex<Vec<AccountAddress>>,
-    write_mutex: AsyncMutex<LogicalTime>,
+    write_mutex: AsyncMutex<LogicalTime>, // TODO: mutex needed?
```

Comment:
There might be a race between commit and sync_to (consensus and state sync) when updating the committed logical time. And this mutex should not affecting the performance at all, so it is better to keep it.

Reply:
Oops will remove the TODO :)